src/containers/flows/MintWheels/labels.tsx

```diff
@@ -82,23 +82,37 @@ export const getBannerLabel = (
 		return (
 			<div style={{ display: 'flex', flexDirection: 'column' }}>
 				<span>
-					Wilder Wheels now available for whitelisted supporters{' '}
+					Wilder Kicks now available for whitelisted supporters{' '}
 					{totalLabel(wheelsMinted!, wheelsTotal!)}
 				</span>
 				{timer}
 			</div>
 		);
 	}
-	if (dropStage === Stage.Public) {
+	// if (dropStage === Stage.Public) {
+	// 	return (
+	// 		<>
+	// 			Minting is now open to everyone, act fast to secure your Kicks!{' '}
+	// 			{totalLabel(wheelsMinted!, wheelsTotal!)}
+	// 		</>
+	// 	);
+	// }
+	if (
+		dropStage === Stage.Public &&
+		Math.max(wheelsTotal! - wheelsMinted! - 3423, 0) !== 0
```

Comment:
What does the 3423 represent here? If it's residual from the Wheels launch, I believe it's there because the first phase of the sale sold 3423, which needed to be deducted from the counter for second sale. Could be wrong there though.

Nit:
Is `Math.max(...)` here just checking if there are still any wheels remaining? May be more readable as `wheelsTotal! - wheelsMinted! - 3423 > 0`.

Reply:
I also have this question and wanted to check with you.